pkg/debug/checkups/launcher_flags.go

```diff
@@ -0,0 +1,85 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+	"github.com/peterbourgon/ff/v3"
+)
+
+type launcherFlags struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+}
+
+func (lf *launcherFlags) Name() string {
+	return "Launcher Flags"
+}
+
+func (lf *launcherFlags) Run(_ context.Context, extraFh io.Writer) error {
+	lf.status = Failing
+
+	configFilePath := lf.flagsFilePath()
+	fmt.Fprint(extraFh, fmt.Sprintf("flags file path: %s\n", configFilePath))
+
+	stat, err := os.Stat(configFilePath)
+	if err != nil {
+		lf.summary = fmt.Sprintf("failed to stat %s: %s", configFilePath, err)
+		return nil
+	}
+
+	fmt.Fprint(extraFh, fmt.Sprintf("flags file is %d bytes, and was last modified at %s\n", stat.Size(), stat.ModTime()))
+
+	file, err := os.Open(configFilePath)
+	if err != nil {
+		lf.summary = fmt.Sprintf("failed to open %s: %s", configFilePath, err)
+		return nil
+	}
+
+	fmt.Fprint(extraFh, "Parsed Flags (boolean flags appear as {KEY} TRUE):\n\n")
+
+	if err := ff.PlainParser(file, func(name, value string) error {
+		// print each flag and value to file handler
+		fmt.Fprint(extraFh, fmt.Sprintf("%s %s\n", name, value))
```

Comment:
This is an interesting approach. Do you know if we lose information vs just copying the file?

Reply:
Maybe both'ish? copy the file, and dump the resultant options through data?

Reply:
now just dumping the file to the log, easy enough